plugins/outputs/bigquery/bigquery.go

```diff
@@ -111,6 +134,20 @@ func (s *BigQuery) Write(metrics []telegraf.Metric) error {
 	return nil
 }
 
+func (s *BigQuery) writeCompact(metrics []telegraf.Metric) error {
+	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(ctx, time.Duration(s.Timeout))
+	defer cancel()
+
+	inserter := s.client.DatasetInProject(s.Project, s.Dataset).Table(s.CompactTableName).Inserter()
```

Comment:
What happens here if the table does not exist?

Reply:
Nothing special, but the Put method will give an error. 

Note I also check the existence of the table in the plugin Connect method.

Reply:
Sure, totally right. I will do an extra check to verify no problems occur.

Reply:
The inserter always returns a `Inserter` struct: https://github.com/googleapis/google-cloud-go/blob/422c0b7565381a5441bc0e8fd81fe7401b961f2a/bigquery/inserter.go#L68-L70